sdk/client/src/index.tsx

```diff
@@ -724,6 +724,8 @@ SessionSecureID: ${this.sessionData.sessionSecureID}`,
 					sampling: {
 						canvas: {
 							fps: this.samplingStrategy.canvas,
+							fpsManual:
```

Comment:
I ran into some type errors here trying to start the client, so wanted to double check that it was a local issue (my assumption since builds are all passing)

```
src/index.tsx:727:8 - error TS2322: Type '{ fps: number | "all" | undefined; fpsManual: number | undefined; resizeFactor: number | undefined; clearWebGLBuffer: boolean | undefined; initialSnapshotDelay: number | undefined; dataURLOptions: { ...; }; maxSnapshotDimension: number | undefined; }' is not assignable to type 'Partial<{ fps: number | "all"; resizeFactor: number; maxSnapshotDimension: number; clearWebGLBuffer?: boolean | undefined; initialSnapshotDelay?: number | undefined; dataURLOptions?: Partial<...> | undefined; }>'.
  Object literal may only specify known properties, and 'fpsManual' does not exist in type 'Partial<{ fps: number | "all"; resizeFactor: number; maxSnapshotDimension: number; clearWebGLBuffer?: boolean | undefined; initialSnapshotDelay?: number | undefined; dataURLOptions?: Partial<...> | undefined; }>'.

727        fpsManual:
           ~~~~~~~~~~
728         this.samplingStrategy.canvasManualSnapshot,
    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

src/index.tsx:1211:16 - error TS2339: Property 'snapshotCanvas' does not exist on type 'typeof record'.

1211   await record.snapshotCanvas(element)
```